components/brave_rewards/browser/rewards_protocol_navigation_throttle.cc

```diff
@@ -92,7 +92,7 @@ GURL TransformUrl(const GURL& url) {
   DCHECK(url.is_valid());
 
   return GURL(base::StrCat(
-      {base::StrCat({"chrome", url::kStandardSchemeSeparator, "rewards/"}),
+      {"chrome", url::kStandardSchemeSeparator, "rewards/", url.host(), "/",
```

Comment:
just wondering how did you find the spots that needed fixing and how to ensure there is nothing similar to fix

Reply:
It relied on brave unit/browser tests, upstream tests, and some spot tests of various components (e.g., IPFS). It's definitely hard to be 100% confident we caught everything, though. I thought it was better to make the attempt than to disable the feature flag, though,